install.sh

```diff
@@ -45,7 +38,7 @@ do
             help=1
         fi
         shift
-    elif [[ ${var} == "--samples" ]] ; then
+    elif [[ $1 == "--samples" ]] ; then
```

Comment:
I suggest to always quote command line arguments like `$1` here and in all other lines. Otherwise it would be an error if whatever argument contains whitespace, e.g.
```sh
$ ./install.sh --extends-workspace "/path/to/Who would ever use spaces in directory names?"
./install.sh: line 34: [: too many arguments
```

Reply:
This also applies to other scripts which accept command line arguments.